tests/sles4sap/qesapdeployment/deploy.pm

```diff
@@ -26,6 +27,9 @@ sub run {
         my $rec_timeout = qesap_ansible_log_find_timeout($ret[1]);
         if ($rec_timeout) {
             record_info('DETECTED ANSIBLE TIMEOUT ERROR');
+            $self->clean_up();
+            @ret = qesap_execute(cmd => 'terraform', verbose => 1, timeout => 1800);
```

Comment:
maybe `$self->{result}` can be used here to decide if or if not to run `terraform`